h2o-py/tests/pyunit_utils/utilsPY.py

```diff
@@ -3643,6 +3643,20 @@ def build_save_model_GLM(params, x, train, respName):
     model.download_mojo(path=TMPDIR)    # save mojo
     return model
 
+def build_save_model_GAM(params, x, train, respName):
+    # build a model
+    model = H2OGeneralizedAdditiveEstimator(**params)
+    model.train(x=x, y=respName, training_frame=train)
+    # save model
+    regex = re.compile("[+\\-* !@#$%^&()={}\\[\\]|;:'\"<>,.?/]")
+    MOJONAME = regex.sub("_", model._id)
+
+    print("Downloading Java prediction model code from H2O")
+    TMPDIR = os.path.normpath(os.path.join(os.path.dirname(os.path.realpath('__file__')), "..", "results", MOJONAME))
+    os.makedirs(TMPDIR)
+    model.download_mojo(path=TMPDIR)    # save mojo
+    return model
```

Comment:
```suggestion
    return model.download_mojo(path=TMPDIR)    # save mojo
```
Shouldn't we return the path returned by the `download.mojo` method ?

Reply:
In this case, I actually need to model back in order to use it to do scoring using the model.  Then, I use mojo to do the scoring and compare the two to make sure they are the same.